### PR TITLE
Update and add triggers

### PR DESCRIPTION
**Pull request by Linux123123**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -26,30 +26,30 @@
...
+    "@types/node": "^16.11.6",
+    "@types/node-fetch": "v2",
     "chalk": "^4.1.2",
     "discord-api-types": "^0.24.0",
-    "discord.js": "^13.1.0",
+    "discord.js": "^13.3.1",
     "dotenv": "^10.0.0",
     "moment": "^2.29.1",
-    "node-fetch": "2.6.2",
+    "node-fetch": "v2",
```
Comment (vilhelmprytz) [inline package.json:36]: What's this? Why just `v2` now?

File: src/config/triggers.ts [hunk 2]
```diff
@@ -68,6 +68,7 @@ export const triggers: Trigger[] = [
       "Gives instructions on how to retrieve logs for the panel and daemon",
     aliases: ["log", "o,l.gs", "lokgs", "logas"],
     keys: [
+      "host to get script logs",
```
Comment (vilhelmprytz) [inline src/config/triggers.ts:71]: Should be `how? 
```suggestion
      "how to get script logs",
```

File: src/events/messageCreate.ts [hunk 3]
```diff
@@ -48,7 +48,8 @@ export const run: RunFunction = async (bot, message: Message) => {
     triggers = triggers.concat(
       bot.functions.findTriggers(message.content.toLowerCase())
     );
-    const RegExp = /(https?:\/\/bin.ptdl.co\/([^\s]+))/g;
+    const RegExp =
+      /(https?:\/\/(bin.ptdl.co\/([^\s]+)|termbin.com\/([^\s]+)))/g;
```
Comment (vilhelmprytz) [inline src/events/messageCreate.ts:52]: Termbin is not really known to me. Do we have to "index" it as well?

File: package.json (hunk 1 above)
Comment (Linux123123) [reply package.json:36]: `node-fetch` v3 is not compatible with our project as it is ES module only...
It was v2 already, but I just made it v2 in `package.json` to really keep it at v2.

https://github.com/node-fetch/node-fetch/blob/main/docs/CHANGELOG.md#v300

v2 still being update tho :)

File: src/events/messageCreate.ts (hunk 3 above)
Comment (Linux123123) [reply src/events/messageCreate.ts:52]: There is no real difference between them. Both are user uploaded content, and that is about it. Just one is by pterodactyl and the other by `termbin`. I think it is the same "risk" level for both of them.

Comment (Linux123123) [reply src/events/messageCreate.ts:52]: There is no real difference between them. Both are user uploaded content, and that is about it. Just one is by pterodactyl and the other by termbin. I think it is the same "risk" level for both of them.

**Pull request merged**